### PR TITLE
Improve logging of AVC check plugin and its test

### PR DESCRIPTION
**Pull request by happz**
Log more selinux-related packages, auditd status, and system journal.

Pull Request Checklist

* [x] implement the feature

### CODE REVIEW

File: tmt/checks/avc.py [hunk 1]
```diff
@@ -121,17 +126,18 @@ def _report_failure(label: str, exc: tmt.utils.RunError) -> list[str]:
             report += _report_failure('sestatus', exc)
 
         # Record selinux-policy NVR.
```
Comment (thrix) [inline tmt/checks/avc.py:128]: This needs update

File: tmt/checks/avc.py [hunk 2]
```diff
@@ -121,17 +126,18 @@ def _report_failure(label: str, exc: tmt.utils.RunError) -> list[str]:
...
-        output, exc = _run_script(ShellScript('rpm -q selinux-policy'))
+        interesting_packages = ' '.join(INTERESTING_PACKAGES)
+        output, exc = _run_script(ShellScript(f'rpm -q {interesting_packages} || /bin/true'))
```
Comment (thrix) [inline tmt/checks/avc.py:130]: with added `|| /bin/true` this script never fails? 

I would rather say both `selinux-policy` and `audit` are require for this check to work properly, so not sure why it should not fail

**Review by thrix — DISMISSED** [review]

File: tmt/checks/avc.py (hunk 2 above)
Comment (psss) [reply tmt/checks/avc.py:130]: Agreed, sounds reasonable. Otherwise the exception handling below would not make any sense.

**Review by psss — CHANGES_REQUESTED** [review]
Just the two minor things mentioned by @thrix.

File: tmt/checks/avc.py (hunk 1 above)
Comment (happz) [reply tmt/checks/avc.py]: Fixed in 18d2e7a8

File: tmt/checks/avc.py (hunk 2 above)
Comment (happz) [reply tmt/checks/avc.py]: Fixed in 18d2e7a8

File: tests/test/check/data/main.fmf [hunk 3]
```diff
@@ -13,6 +13,8 @@
   /nasty:
     test: |
       set -x
+      systemctl status auditd || /bin/true
       sudo bash -c "passwd --help &> /root/passwd.log; \
                     ls -alZ /root/passwd.log; \
                     rm -f /root/passwd.log" || /bin/true
+      journalctl -b --no-pager
```
Comment (psss) [inline tests/test/check/data/main.fmf:20]: This produces over 2000 lines of output. Do we really need the whole content? If so, what about storing it in a dedicated file under `$TMT_TEST_DATA` ? If not, what about using a more recent timestamp?

Comment (happz) [reply tests/test/check/data/main.fmf]: Dropped in 579003ec

Comment (psss) [reply tests/test/check/data/main.fmf]: Good!

**Review by psss — APPROVED** [review]

**Pull request merged**